docs/_includes/head.html

```diff
@@ -23,5 +23,5 @@
 
     <!-- CSS -->
     <link rel="stylesheet" href="//cdn.jsdelivr.net/npm/docsearch.js@2/dist/cdn/docsearch.min.css" />
-    <link rel="stylesheet" type="text/css" href="{{ "/assets/css/stacks-documentation.min.css" | relative_url }}?{{page.date | date: '%s%N'}}">
+    <link rel="stylesheet" type="text/css" href="{{ "/assets/dist/stacks.min.css" | relative_url }}?{{page.date | date: '%s%N'}}">
```

Comment:
Just a heads up, the way the docs work now is the stacks bundle + whatever's in `stacks-documentation.less`. If I'm not mistaken, this loses any custom docs CSS.

Reply:
Nope! The file is the same, just the name changed. By default, the style bundles take the same name as the JS bundles, but this can be easily changed. Since the JS file name changed to `stacks.js`, the css file name changed as well. You can see [here](https://github.com/StackExchange/Stacks/pull/707/files#diff-6760708bea0e9d8fe9c272531d931fcf8fd7424db4de10dcba92031a508bcc13R3) that I'm including the correct css file.